### PR TITLE
ability to track date site adopted

### PR DESCRIPTION
**Pull request by vrushalitarte**
Implemented ability to track date adopted
-Added date_adopted field to the adopted_sites table 
-Updated the Adopt a Site route to set the date_adopted to the current date

### CODE REVIEW

File: service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java [hunk 1]
```diff
@@ -130,9 +130,9 @@ public UsersRecord createNewUser(
   public void addToBlackList(String signature) {
     Timestamp expirationTimestamp = Timestamp.from(Instant.now().plusMillis(msRefreshExpiration));
     db.insertInto(Tables.BLACKLISTED_REFRESHES)
-            .values(signature, expirationTimestamp)
-            .onDuplicateKeyIgnore()
-            .execute();
+        .values(signature, expirationTimestamp)
```
Comment (willmt80) [inline service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java:133]: Was this automatic from spotless:apply?

**Review by willmt80 — COMMENTED** [review]
Looks good to me! Waiting to get approval from Conner about migration script

File: persist/src/main/resources/db/migration/V3_5__AddDateAdopted.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+ALTER TABLE adopted_sites ADD COLUMN date_adopted DATE DEFAULT NULL;
```
Comment (willmt80) [inline persist/src/main/resources/db/migration/V3_5__AddDateAdopted.sql:1]: Could you rename this V3_6__AddDateAdopted.sql? I already migration changes and neighborhood importing with my migration script which is V3_5.

**Review by willmt80 — CHANGES_REQUESTED** [review]

File: service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java (hunk 1 above)
Comment (connernilsen) [reply service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java:133]: Good catch. Double check by running `mvn spotless:apply` @vrushalitarte

**Review by connernilsen — APPROVED** [review]
Good work! Just address the comments above and you should be good!

File: service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java (hunk 1 above)
Comment (vrushalitarte) [reply service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java:133]: yes it was automatic

**Review by adriannaliu — APPROVED** [review]

File: persist/src/main/resources/db/migration/V3_5__AddDateAdopted.sql (hunk 2 above)
Comment (vrushalitarte) [reply persist/src/main/resources/db/migration/V3_5__AddDateAdopted.sql]: done

File: persist/src/main/resources/db/migration/V3_6__AddDateAdopted.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+ALTER TABLE adopted_sites ADD COLUMN date_adopted DATE DEFAULT NULL;
```
Comment (willmt80) [inline persist/src/main/resources/db/migration/V3_6__AddDateAdopted.sql:1]: 👍

**Review by willmt80 — APPROVED** [review]

**Pull request merged**